recipes/model_metadata/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "model_metadata" %}
+{% set version = "0.3.3" %}
+{% set build = "0" %}
```

Comment:
If you are using conda-skeleton to generate your recipe please updated to the latest version. We are moving away for the excess of jinja variables and conda-skeleton should do the right thing for you there.